--- scribe/src/store.cpp.orig	2011-05-12 17:18:53.901689379 +1000
+++ scribe/src/store.cpp	2011-05-12 17:20:23.184368643 +1000
@@ -578,7 +578,12 @@
 
     // this is the case where there's no file there and we're not incrementing
     if (suffix < 0) {
-      suffix = 0;
+      if (rollPeriod == ROLL_HOURLY) {
+        suffix = current_time->tm_hour;
+      }
+      else {
+        suffix = 0;
+      }
     }
 
     string file = makeFullFilename(suffix, current_time);
